From 8a04a8345f78ab8f916d824c33deb24e4cf1a80a Mon Sep 17 00:00:00 2001
From: Stephen Gordon <gordoste@iinet.net.au>
Date: Mon, 23 Dec 2024 10:01:31 +1100
Subject: [PATCH] configs: Enable PCM3168A driver

The ezsound 6x8 soundcard requires the PCM3168A codec driver with
I2C support.

Signed-off-by: Stephen Gordon <gordoste@iinet.net.au>
---
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcm2712_defconfig | 1 +
 2 files changed, 2 insertions(+)

--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1152,6 +1152,7 @@ CONFIG_SND_SOC_CS4265=m
 CONFIG_SND_SOC_ICS43432=m
 CONFIG_SND_SOC_MA120X0P=m
 CONFIG_SND_SOC_MAX98357A=m
+CONFIG_SND_SOC_PCM3168A_I2C=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_TLV320AIC23_I2C=m
 CONFIG_SND_SOC_WM8804_I2C=m
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -1154,6 +1154,7 @@ CONFIG_SND_SOC_CS4265=m
 CONFIG_SND_SOC_ICS43432=m
 CONFIG_SND_SOC_MA120X0P=m
 CONFIG_SND_SOC_MAX98357A=m
+CONFIG_SND_SOC_PCM3168A_I2C=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_TLV320AIC23_I2C=m
 CONFIG_SND_SOC_WM8804_I2C=m
